packages/components/radio/src/Radio.variants.ts

```diff
@@ -0,0 +1,46 @@
+import { cva, VariantProps } from 'class-variance-authority'
+
+const defaultVariants = {
+  intent: 'primary',
+  size: 'small',
+} as const
+
+export const radioVariants = cva(
+  [
```

Comment:
Maybe we could regroup CSS classes by category, instead of having only one class per item in the array 😁,
 ```ts
[
   'block rounded-full border-md...',
   'hover:outline hover:outline-4....',
   'disabled:border-oultine...'
 ]
```